rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -101,7 +111,36 @@ public BookKeeper(Configuration conf, MetricRegistry metrics) throws FileNotFoun
    */
   private void initializeMetrics()
   {
+    cacheEvictionCount = metrics.counter(METRIC_BOOKKEEPER_CACHE_EVICTION_COUNT);
     localCacheCount = metrics.counter(METRIC_BOOKKEEPER_LOCAL_CACHE_COUNT);
```

Comment:
rename `localCacheCount` to `totalRequestCount` and `localRequestCount` to `cacheRequestCount`

Reply:
@abhishekdas99 Renamed